src/components/Navbar/Navbar.tsx

```diff
@@ -39,11 +39,12 @@ const Navbar = (props: Props) => {
 
   const getNavListLink = (link: NavLink, index: number) => (
     <NavDropdown.Item
-      className={link.className}
+      className={link.dividerAbove ? 'border-top mt-1 pt-2' : ''}
       href={link.href ? link.href : ''}
       key={index}
       onClick={link.onClick}
     >
+      {link.icon && <Icon icon={link.icon} className="fa-fw mr-3" />}
```

Comment:
`icon={list.name as IconType}` because the icon prop of the Icon component only takes `IconType`

Reply:
`icon={list.name as IconType}`

Do you mean `link.icon` instead of `list.name`, there?   There is no `list` in getNavListLink, only `link`.

Assuming it's `link.icon`, that also solves the  `icon?: any` issue in interfaces mentioned above.  I originally put it as type `any` as it was rejecting it as a string...but passing it with `as IconType` as you suggest has resolved that.  I can see now why it was rejecting the string, thank you!